src/main/kotlin/com/autonomousapps/tasks/ReasonTask.kt

```diff
@@ -253,6 +253,29 @@ abstract class ReasonTask @Inject constructor(
       }
 
     private fun wasFiltered(): Boolean = finalAdvice == null && unfilteredAdvice != null
+
+    internal companion object {
+      internal fun findFilteredDependencyKey(dependencies: Set<Map.Entry<String, Any>>, requestedId: String): String? {
```

Comment:
does this need to be internal instead of private?

Reply:
because of accessing from the unit test